app/Http/Controllers/WordController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use App\Models\Word;
+use Illuminate\Http\Request;
+
+class WordController extends Controller
+{
+
+    public function getUserWords(Request $request, int $user_id)
+    {
+        $user = User::find($user_id);
+        if (!$user)
+        {
+            return response("User $user_id not found", 404);
+        }
+
+        $words = Word::where('user_id', $user_id)->get();
+        if (count($words) == 0)
+        {
+            return response("No words found for user $user_id", 404);
+        }
+
+        //placeholder for now; words will always be viewed on a board, right? we wont have stand alone words?
+        $word_list = $words->map(function ($word) {
```

Comment:
all the ORM models (and collections of ORM models, like what `get()` on line 20 returns) automatically implement a `toArray()` function. you can also control what gets put into the JSON with a `$visible` attribute like [here](https://github.com/peeley/lucy/blob/55886e82db637d3188826950ff8ef972f2e40999/app/Models/Word.php#L25). so you could actually save yourself a bit of code and just do `$word_list = $words->toArray();` here.